prototype/storage_api.py

```diff
@@ -0,0 +1,47 @@
+import mmif
+from fastapi import FastAPI
+from enum import Enum
+from pydantic import BaseModel
+from typing import List, Dict
+import os
+import json
+
+app = FastAPI()
+# get post request from user
+# read mmif inside post request, get view metadata
+# store in nested directory relating to view metadata
+
+# TODO: there's probably a better structure for this. Not sure if this works at the moment
+# TODO: but either way should hopefully be changed to utilize mmif operations?
+class Item(BaseModel):
```

Comment:
I don't really think that we need to define a pydantic model of the input MMIF (see the link bottom of this tutorial https://fastapi.tiangolo.com/tutorial/body/ ), but instead just use validation function in the MMIF sdk. 
(https://clams.ai/mmif-python/latest/autodoc/mmif.serialize.html#mmif.serialize.mmif.Mmif)